src/libstd/io/buffered.rs

```diff
@@ -182,6 +206,10 @@ impl<R: Read> Read for BufReader<R> {
     }
 }
 
+// we cant impl unconditionally because of the large buffer case in read.
+#[unstable(feature = "trusted_len", issue = "0")]
```

Comment:
Is this supposed to be "trusted_read"?  "trusted_len" also appears in a few other places.